enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+trust the mirror registry.  We see this problem w/ imagestream import which expects to get its CAs 
+from an image config resource which references a configmap of bundles that are explicitly 
+associated to registry hostnames.  Further complicating the situation is that the field is simply 
+named `additionalTrustBundle` so the user intent is not clear, making it difficult for specific 
+components to know whether they should be consuming this value or not.
+
+The most direct consequences of this are that today if a customer does a disconnected install, 
+their imagestream imports from the mirror registry will fail.  In particular this means the 
+oauth-proxy imagestream, which is consumed by many OLM operators, will fail and thus those 
+operators will fail to function, until/unless the admin explicitly configures the CA in the image 
+config resource and then re-imports the imagestream.
+
+While this could theoretically be solved by the imagestream component trusting the additional CA (
+for all hosts?) explicitly, it would make more sense if we used the information the admin is 
+providing us at install time to correctly configure the cluster as if they had done it themselves, 
+rather than introducing additional sources of (potentially conflicting) configuration 
+information.
+
+## Motivation
+
+See summary above, motivation is to make the install experience clearer with less potential for misconfiguration and conflicting settings.
+
+### Goals
+
+- Post install the cluster can not only pull images from an alternate registry, but imagestreams can import from it, builds can pull from it, and the internal registry can perform pullthrough against it
+- Post install the registry CAs provided by the admin are populated into the configuration api that is intended for this purpose
+
+### Non-Goals
+
+- Refactoring the additionalTrustBundle field into many unique fields for different CA usages
+- Addressing the current proxy CA behavior in which the additionalTrustBundle is sometimes populated into a proxy configuration and other times not (depending on whether a proxy host is supplied)
+- Adding generic cluster configuration fields to the installer that are not required to bootstrap the cluster and ensure consistent use of cluster config apis
+
+
+## Proposal
+
+The installer should populate the 
+[image config resource](https://github.com/openshift/api/blob/master/config/v1/types_image.go) in the same way the 
+[product documentation](https://docs.openshift.com/container-platform/4.5/openshift_images/image-configuration.html#images-configuration-cas_image-configuration) 
+recommends.  The installer 
+can use the value of the additionalTrustBundle field to populate this api.  Similar to how the 
+value is used for the proxy config when a proxy host is supplied, the value can be used to 
+populate the image config resource when a mirror registry is in use.
+
```

Comment:
Can we include

- how the Image object should look like?
- when the installer should be configuring the Image object?
  when imageContentSources are set in the install-config that provides the local registry mirrors for content maybe?
  or always when the additionalTrustBundle is set?

Reply:
yes, will provide an example of the image config resource.

as for when (i guess you mean under what conditions, not what time sequence), yes i think we can limit it to "just when an imageconfigsource is provided/set" but i'll think about it a little more.

Reply:
on further reflection, since the image CA config api requires a registry hostname, it's actually probably impossible to configure the image config resource with the installer CA bundle unless an imagecontentsource is provided(since that's where we'd get the registry hostname from).